src/pybind/mgr/dashboard_v2/models/nodb.py

```diff
@@ -821,7 +822,10 @@ def handle_field(field):
         return model_args
 
     def __init__(self, *args, **kwargs):
-        self._state = mock.Mock()
```

Comment:
You must also remove the `from mock import mock` line in the beginning of the file.

Reply:
fixed